src/_includes/page-footer.html

```diff
@@ -37,8 +37,7 @@ <h4>Technologies</h4>
           <ul>
             <li><a href="{{site.flutter}}">Flutter</a></li>
```

Comment:
Should the text be "Mobile (Flutter)" to match the sidenav?

Reply:
I also changed the order in the sidenav, to match the order of the footer and of the Install and Platforms pages.